pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -102,7 +115,7 @@ export default Vue.extend<Data, Methods, Computed, never>({
         @close="snapshotEvent=null"
       >
         <span
-          v-clean-html="t(`snapshots.info.${ snapshotEvent.type }.${ snapshotEvent.result }`, { snapshot: snapshotEvent.snapshotName, error: snapshotEvent.error }, true)"
+          v-clean-html="t(`snapshots.info.${ snapshotEvent.type }.${ snapshotEvent.result }`, { snapshot: escapeSnapshotName(snapshotEvent.snapshotName), error: snapshotEvent.error }, true)"
```

Comment:
This line is getting a bit long; please consider line wrapping.

Reply:
wrapped it